sdk/lib/crt/math/arm/fabs.s

```diff
@@ -1,24 +0,0 @@
-/*
- * COPYRIGHT:         BSD - See COPYING.ARM in the top level directory
- * PROJECT:           ReactOS CRT library
- * PURPOSE:           Implementation of fabs
- * PROGRAMMER:        Timo Kreuzer (timo.kreuzer@reactos.org)
- */
-
-/* INCLUDES ******************************************************************/
-
-#include <kxarm.h>
-
-/* CODE **********************************************************************/
-
-    TEXTAREA
-
-    LEAF_ENTRY fabs
-
-	__assertfail
-	bx	lr
```

Comment:
@binarymaster / @DarkFire01 would be interested by that :)

Side-question: Is the portable C implementation you've added is going to be used on ARM?

Reply:
Check the cmake file?